CHANGELOG_PENDING.md

```diff
@@ -17,6 +17,7 @@
 ### Bug Fixes 🐞
 
 #### General
+- \#2299 Prevent Transcoder/Broadcaster run scripts from using same CLI port
```

Comment:
I would rephrase it, because:
1. It's not longer a bug fix, it's actually a feature (to enable both split O/T and combined OT setup in devtool)
2. I'd mentioned that it's about devtool